src/main/scala/is/hail/methods/VariantQC.scala

```diff
@@ -112,23 +112,23 @@ final class VariantQCCombiner {
 
 object VariantQC {
   val signature = TStruct(
-    "callRate" -> TFloat64(),
+    "call_rate" -> TFloat64(),
     "AC" -> TInt32(),
     "AF" -> TFloat64(),
-    "nCalled" -> TInt32(),
-    "nNotCalled" -> TInt32(),
-    "nHomRef" -> TInt32(),
-    "nHet" -> TInt32(),
-    "nHomVar" -> TInt32(),
-    "dpMean" -> TFloat64(),
-    "dpStDev" -> TFloat64(),
-    "gqMean" -> TFloat64(),
-    "gqStDev" -> TFloat64(),
-    "nNonRef" -> TInt32(),
-    "rHeterozygosity" -> TFloat64(),
-    "rHetHomVar" -> TFloat64(),
-    "rExpectedHetFrequency" -> TFloat64(),
-    "pHWE" -> TFloat64())
+    "n_called" -> TInt32(),
```

Comment:
Do you think it's inconsistent to have AC and AF all in capitals while the rest is all lower case?

Reply:
I think that's reasonable, but let's leave that for a separate discussion because I imagine there might be some feelings.